sdk/src/main/java/io/opentelemetry/sdk/trace/Samplers.java

```diff
@@ -63,6 +63,20 @@
   // No instance of this class.
   private Samplers() {}
 
+  /**
+   * Returns a {@link Decision} with empty attributes and {@link Decision#isSampled()} returning the
+   * value of the parameter {@code isSampled}.
+   *
+   * <p>This is meant for use by custom {@link Sampler} implementations.
+   *
+   * @param isSampled The value to return from {@link Decision#isSampled()}.
+   * @return A {@link Decision} with empty attributes and {@link Decision#isSampled()} returning the
+   *     value of the parameter {@code isSampled}.
+   */
+  public static final Decision emptyDecision(boolean isSampled) {
```

Comment:
What about having just a `decision(bool, map)` and you would call that with a `emptyMap`?

```
public static final Decision decision(boolean isSampled, Map<String, AttributeValue> attributes) {
  Objects.requireNotNull(attributes, "attributes");
  if (attributes.isEmpty()) {
    return isSampled ? EMPTY_SAMPLED_DECISION : EMPTY_NOT_SAMPLED_DECISION;
  }
  return DecisionImpl.create(isSampled, attributes);
}
```

Reply:
Ah, the DecisionImpl was not there when I created the PR I guess 😃  That totally makes sense.

Reply:
I added `Samplers.decision` but decided to keep `emptyDecision`, since using the explicit empty map is quite a mouthful (CheckStyle even requires a parameter comment for the sampled boolean) and it is needed in the implementation anyway.

Reply:
What about calling this `decision` as well?

Reply:
I'm fine with either way. Would you prefer that?